src/quacc/runners/ase.py

```diff
@@ -309,6 +326,81 @@ def run_vib(self, vib_kwargs: VibKwargs | None = None) -> Vibrations:
 
         return vib
 
+    def run_md(
+        self,
+        timestep: float = 1.0,
+        steps: int = 1000,
+        dynamics: MolecularDynamics = VelocityVerlet,
+        dynamics_kwargs: dict[str, Any] | None = None,
+        initial_temperature: float | None = None,
+        fix_com: bool = True,
+        fix_rot: bool = True,
```

Comment:
These were set to `True` by default in your commits. Is this standard? If someone sets `initial_temperature`, it's expected that COM and rotation are also fixed? My naive assumption would be that this would be unexpected, but I'm not an MD guy so wanted to check before touching this.